### PR TITLE
Inclusion of Geological Times form ICS Chart

### PR DESCRIPTION
**Pull request by alexandretessarollo**
Geological time terms based on ICSchart
RDF file: https://github.com/alexandretessarollo/timescale-data/blob/master/rdf/isc2019.ttl
JPG file: http://www.stratigraphy.org/ICSchart/ChronostratChart2020-01.jpg

### CODE REVIEW

File: src/noun.time [hunk 1]
```diff
@@ -238,39 +244,11 @@ d: the first of the four Sundays during Advent
 hp: Advent[1]
 hypo: Christian_holy_day
 
-w: Age_of_Fishes
```
Comment (arademaker) [inline src/noun.time:241]: I don't like the idea of removing this. See wikipedia: "Fish reached substantial diversity during this time, leading the Devonian to often be dubbed the Age of Fishes"

I would not remove the lexical entries (Age of Fishes) nor the additional observataions in the glosses.

Comment (arademaker) [reply src/noun.time:241]: same for all other cases where you remove MWEs and reduce glosses of `popular` names for the periods.

**Comment by arademaker** [comment]
See my comments. In line with the papers from the Medical Wordnet. We have to remember to keep in the resource terms and descriptions for a layperson too. This is not a tentative of making PWN only for specialists.

File: src/noun.time (hunk 1 above)
Comment (alexandretessarollo) [reply src/noun.time:241]: Solved for all cases in last commit.

Checked all the information in the previous/original glosses against wikipedia. Somo were correct (and brought back), some were incorrect (and kept out) and some incomplete (and improved).

Also brought back the other names (synonyms) where they Wikipedia validation was available.

**Comment by alexandretessarollo** [comment]
> See my comments. In line with the papers from the Medical Wordnet. We have to remember to keep in the resource terms and descriptions for a layperson too. This is not a tentative of making PWN only for specialists.

Medical WordNet attempted to create a corpus of sentences from medical facts (necessarily correct) and another for medical beliefs (which a layperson believe is true, but it may or may not be true).

With the last commit I had in mind to keep the geological fact (correct ones) and bring back only the correct beliefs. Specialist only or not, I don't think WN should keep incorrect terms/definitions/relations.

**Pull request merged**